data/campaigns/Heir_To_The_Throne/utils/httt_utils.cfg

```diff
@@ -408,6 +415,9 @@ fire:  +10%"
         name=moveto
         [filter]
             id="Li'sar"
+            [filter_location]
+                location_id="sceptre"
+            [/filter_location]
```

Comment:
Oh, this also reminds me, you should check if there are instances of this key in the "Sceptre of Fire" campaign too and remove them if they're not needed.